oomact_ros/include/aslam/calibration/ros/InputFeederTemplates.h

```diff
@@ -0,0 +1,37 @@
+#ifndef HB94494AD_043E_41B9_B842_1BDA520FD393
```

Comment:
what is your naming convention for headers!?

Reply:
This is a bit of a mess. I gave up a bit on this for some time because of too much legacy code. And some kind of cleanup regarding this has yet to come. I'm very much open to suggestions there. I'll make it a todo.